samples/wifi/shell/prj.conf

```diff
@@ -96,3 +96,4 @@ CONFIG_SETTINGS_NVS=y
 # design in net_mgmt. So, use a higher timeout for a crowded
 # environment.
 CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
+CONFIG_NET_CONTEXT_PRIORITY=y
```

Comment:
Could you give context for this change.

Reply:
oops, I was testing other feature and this crept in.